modules/ROOT/pages/sending-api-analytics-from-arm-to-db.adoc

```diff
@@ -10,29 +10,39 @@ image:logo-hybrid-active.png[link="/runtime-manager/deployment-strategies", titl
 image:logo-server-active.png[link="/runtime-manager/deployment-strategies", title="Anypoint Platform Private Cloud Edition"]
 image:logo-pcf-disabled.png[link="/runtime-manager/deployment-strategies", title="Pivotal Cloud Foundry"]
 
-The HTTP API analytics produced by applications running on the API gateway runtime can be easily stored in a database by configuring a few settings.
+You can configure Runtime Manager agent to store HTTP API analytics data from Mule apps running on the API gateway runtime in a database.
```

Comment:
That's one tall drink of water of a sentence!
Can you reword a bit to clarify what "in a database" modifies: "store" or "runtime"? "...to store in a database HTTP API analytics data..."?